[2021-12-17 15:33:39,114] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_C manual__2021-12-17T14:33:24.198229+00:00 [queued]>
[2021-12-17 15:33:39,133] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_C manual__2021-12-17T14:33:24.198229+00:00 [queued]>
[2021-12-17 15:33:39,135] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-17 15:33:39,135] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-12-17 15:33:39,135] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-17 15:33:39,152] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): training_model_C> on 2021-12-17 14:33:24.198229+00:00
[2021-12-17 15:33:39,155] {standard_task_runner.py:52} INFO - Started process 12844 to run task
[2021-12-17 15:33:39,161] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'my_dag', 'training_model_C', 'manual__2021-12-17T14:33:24.198229+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp6_2e3mck', '--error-file', '/tmp/tmpuw4j6pep']
[2021-12-17 15:33:39,161] {standard_task_runner.py:77} INFO - Job 14: Subtask training_model_C
[2021-12-17 15:33:39,196] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.training_model_C manual__2021-12-17T14:33:24.198229+00:00 [running]> on host sofienkhanfir-VirtualBox
[2021-12-17 15:33:39,272] {taskinstance.py:1427} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_C
AIRFLOW_CTX_EXECUTION_DATE=2021-12-17T14:33:24.198229+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-17T14:33:24.198229+00:00
[2021-12-17 15:33:39,274] {python.py:152} INFO - Done. Returned value was: 2
[2021-12-17 15:33:39,300] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_C, execution_date=20211217T143324, start_date=20211217T143339, end_date=20211217T143339
[2021-12-17 15:33:39,348] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-17 15:33:39,381] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
